.github/workflows/ci.yml

```diff
@@ -117,3 +117,21 @@ jobs:
       with:
         files: ${{ matrix.covname }}
         fail_ci_if_error: true
+    - name: Installing Cargo-Cache
+      if: ${{ matrix.push }}
+      uses: actions-rs/cargo@844f36862e911db73fe0815f00a4a2602c279505 # v1.0.3
+      with:
+        command: install
+        args: cargo-cache
+    - name: Cleaning Cargo cache
+      if: ${{ matrix.push }}
+      uses: actions-rs/cargo@844f36862e911db73fe0815f00a4a2602c279505 # v1.0.3
+      with:
+        command: cache
+        args: -e -i
+    - name: Getting sccache size
+      if: ${{ matrix.push }}
+      uses: actions-rs/cargo@844f36862e911db73fe0815f00a4a2602c279505 # v1.0.3
+      with:
+        command: cache
+        args: sccache
```

Comment:
Do we actually need sscache?

Reply:
not really? but it's not what is causing the cache to grow, its `/.cargo/registry`

Reply:
it does improve compile times by a bit and stays around 500MB, so I wouldn't remove it just because